edb/server/bootstrap.py

```diff
@@ -463,37 +446,93 @@ async def _make_stdlib(testmode: bool, global_ids) -> StdlibBits:
     )
 
 
+async def _amend_stdlib(
+    ddl_text: str,
+    stdlib: StdlibBits,
+) -> Tuple[StdlibBits, str]:
+    schema = stdlib.stdschema
+    reflschema = stdlib.reflschema
+
+    topblock = dbops.PLTopBlock()
+    plans = []
+
+    context = sd.CommandContext()
+    context.stdmode = True
+
+    for ddl_cmd in edgeql.parse_block(ddl_text):
```

Comment:
I assume this `for` is copy/pasted from `compiler.py`, no? If so I'd factor it out into a helper function in server/compiler.py

Reply:
No, it's not copy/pasted from `compiler.py`, there are important differences from normal processing of DDL.